README.md

```diff
@@ -58,17 +78,78 @@ kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
+```bash
+helm repo update
+helm install stable/nginx-ingress \
+--version 0.8.11 \
+--set controller.scope.enabled=true \
+--set controller.scope.namespace=$DESIREDNAMESPACE \
+--namespace $DESIREDNAMESPACE
+```
+
+### 2. Get the nginx-ingress-controller release name from the previous command and set it as a varible:
+```bash
+export INGRESSRELEASE=knobby-wolf
+```
+
+### 3. Wait for the nginx-ingress-controller release to get deployed (When checking status your pod should be READY 1/1):
+```bash
+helm status $INGRESSRELEASE
+```
+
+### 4. Get the nginx-ingress-controller port for the infrastructure (**NOTE! ONLY FOR MINIKUBE**):
+```bash
+export INFRAPORT=$(kubectl get service $INGRESSRELEASE-nginx-ingress-controller --namespace $DESIREDNAMESPACE -o jsonpath={.spec.ports[0].nodePort})
+```
+
+### 5. Get Minikube or ELB IP and set it as a variable for future use:
+
+```bash
+#ON MINIKUBE
+export ELBADDRESS=$(minikube ip)
+
+#ON AWS
+export ELBADDRESS=$(kubectl get services $INGRESSRELEASE-nginx-ingress-controller --namespace=$DESIREDNAMESPACE -o jsonpath={.status.loadBalancer.ingress[0].hostname})
+```
+
+### 6. EFS Storage (**NOTE! ONLY FOR AWS!**)
+
+Create a EFS storage on AWS and make sure it is in the same VPC as your cluster. Make sure you open inbound traffic in the security group to allow NFS traffic. Save the name of the server ex:
+```bash
+export NFSSERVER=fs-d660549f.efs.us-east-1.amazonaws.com
+```
+
+***Note!***
+The Persistent volume created to store the data on the created EFS has the ReclaimPolicy set to Recycle.
+This means that by default, when you delete the release the saved data is deleted automatically.
+
+To change this behaviour and keep the data you can set the alfresco-infrastructure.persistence.reclaimPolicy value to Retain.
+For more Information on Reclaim Policies checkout the official K8S documentation here -> https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaim-policy
+
+We don't advise you to use the same EFS instance for persisting the data from multiple dbp deployments.
+
+### 7. Add the Alfresco Kubernetes repository to helm.
+
+```helm repo add alfresco-incubator http://kubernetes-charts.alfresco.com/incubator```
+
+### 8. Deploy the DBP
 
 ```bash
 
 #On MINIKUBE
 helm install alfresco-incubator/alfresco-dbp \
+--set alfresco-infrastructure.alfresco-api-gateway.keycloakURL="http://$ELBADDRESS:$INFRAPORT/auth/" \
 --set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://$ELBADDRESS:$INFRAPORT/auth" \
 --namespace=$DESIREDNAMESPACE
 
 #On AWS
 helm install alfresco-incubator/alfresco-dbp \
+--set alfresco-infrastructure.alfresco-api-gateway.keycloakURL="http://$ELBADDRESS/auth/" \
+--set alfresco-infrastructure.persistence.efs.enabled=true \
+--set alfresco-infrastructure.persistence.efs.dns="$NFSSERVER" \
 --set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://$ELBADDRESS/auth" \
 --namespace=$DESIREDNAMESPACE
 ```
```

Comment:
@svidrascu  - we are missing instruction about persistence subpath i think .. not sure

Reply:
We are not setting those in the default readme, just in the deploy script. The subpaths are already set.